data/coalition/coalition weapons.txt

```diff
@@ -77,6 +77,45 @@ effect "bombardment impact"
 	"velocity scale" -.2
 
 
+outfit "Finisher Maegrolain"
+	category "Secondary Weapons"
+	licenses
+		Heliarch
+	cost 7260000
+	thumbnail "outfit/finisher maegrolain"
+	"mass" 40
```

Comment:
Outfit space used = 88
Finisher capacity = 120
Finisher mass = 0.5
120 * 0.5 = 60
88 - 60 = 28
```suggestion
	"mass" 28
```

Reply:
Didn't know there was a calc for it. Thanks.